lib/Runtime/Library/JavascriptFunction.cpp

```diff
@@ -139,7 +139,10 @@ namespace Js
     static char16 const funcName[] = _u("function anonymous");
     static char16 const genFuncName[] = _u("function* anonymous");
     static char16 const asyncFuncName[] = _u("async function anonymous");
-    static char16 const bracket[] = _u(" {\012");
+    static char16 const openFormals[] = _u("(");
+    static char16 const closeFormals[] = _u("\012)");
```

Comment:
We could and probably should.  It's just extra 'trivial' work.  I'm always suspicious of extra 'trivial' work is all.